flytekit/remote/remote.py

```diff
@@ -397,3 +380,61 @@ def _(self, entity: LaunchPlan, inputs: typing.Dict[str, typing.Any], execution_
         except Exception:
             flyte_launchplan: FlyteLaunchPlan = self.register(entity)
         return self.execute(flyte_launchplan, inputs, execution_name)
+
+    @singledispatchmethod
+    def wait_for_completion(
```

Comment:
would this be less clear if we called this `wait`? It's more concise and conveys the same meaning